addon/components/sl-textarea.js

```diff
@@ -104,11 +92,11 @@ export default Ember.Component.extend( InputBased, TooltipEnabled, ComponentInpu
     /**
      * The `spellcheck` HTML attribute value
      *
-     * Accepted values are true, false, "default" (default), "true", or "false".
+     * Accepted values "false" (default) or "true".
      *
-     * @type {Spellcheck}
+     * @type {Boolean}
      */
-    spellcheck: Spellcheck.Direction,
+    spellcheck: false,
```

Comment:
The type is specified to be Boolean so there is no need to handle string values, its not our fault that users pass in a wrong type, because our documentation states a Boolean value. we should not be testing for every single contingency. I tested a computed value called spellCheckString that converted string false to boolean false and vice versa like how the input based mixin does with the disabled property https://github.com/softlayer/sl-ember-components/blob/EmberCli-1.13.8/addon/mixins/sl-input-based.js#L98 , however that is not correct in this case.